search/elastic.py

```diff
@@ -709,6 +713,16 @@ def search(self,
         _sort_args_in_body = kwargs.pop('sort', None)
         if _sort_args_in_body:
             body['sort'] = _sort_args_in_body
+        # Get `doc_count` from ES ( without filters )
+        # E.g: if we query courses with lots of conditions, then we still return total count of
+        # courses with this Flag `ga_total`
+        ga_total = kwargs.pop('ga_total', None)
+        if ga_total:
+            body['aggs'] = {
+                "total_records": {
+                    "global": {}
```

Comment:
- When query ES data with filters, count and return `doc_count` from ES ( `without filters` )